localization/german/opinions/wc_general_opinions_l_german.yml

```diff
@@ -1,24 +1,24 @@
 ﻿l_german:
- being_undead_crime:0 "[GetTrait('being_undead').GetName( GetNullCharacter )]"
- being_undead_intolerant:0 "[GetTrait('being_undead').GetName( GetNullCharacter )]"
- 
- loyal_undead:0 "Loyal[ROOT.Char.CustomLoc_DE_END_ADJ_NUL_S_N] [GetTrait('being_undead').GetName( GetNullCharacter )]"
- attempted_to_kill_me:0 "Hat versucht mich zu töten"
+  being_undead_crime:0 "[GetTrait('being_undead').GetName( GetNullCharacter )|V]"
```

Comment:
Noticed one weird thing. There are 2 spaces at the beginning of the lines in this file. It's not a bug but it's an extra change left from the merge.